bedrock/firefox/templates/firefox/new/fx/scene1.html

```diff
@@ -80,5 +80,8 @@ <h2>Need to download again?</h2>
 {% endblock %}
 
 {% block js %}
+  {% if switch('stub-attribution-macos') %}
```

Comment:
This experiment is finished and was shown only to existing Firefox users, is this attribution code something we need to include here? (one less thing to manually test and verify if the answer is no)